fury/ui.py

```diff
@@ -5240,3 +5240,165 @@ def _set_position(self, coords):
         pass
         # self.actor.SetPosition(*coords)
         # self.container.SetPosition(*coords)
+
+
+class Card2D(UI):
+
+    """Card element to show image and related text
+    """
+
```

Comment:
Please add the `Attributes` section here which explains what fundamental UI elements are used to render this particular UI component. Look at the `ComboBox2D` code for e.g.